frontend/server/src/DAO/Problems.php

```diff
@@ -828,20 +828,44 @@ public static function getAdminUser(\OmegaUp\DAO\VO\Problems $problem): ?array {
         ];
     }
 
+    /**
+     * @return list<\OmegaUp\DAO\VO\Problems>
+     */
+    public static function getAllWithTotal(
```

Comment:
```suggestion
    public static function getAllWithCount(
```

a MySQL le pides el `COUNT(*)`, entonces `WithCount` es más consistente.

Reply:
Listo